File: problem-set/Problem-Petya-And-Strings/Petya-And-Strings.js
function comapreTwoStrings(str1, str2) {
  let num1 = 0;
  let num2 = 0;

  for (x = 0; x < str1.length; x++) {
    num1 = str1[x].toLowerCase();
    num2 = str2[x].toLowerCase();

    if (num1 < num2) {
      return -1;
    } else if (num1 > num2) {
      return 1;
    } else {
      return 0;
    }
  }
}

console.log(comapreTwoStrings("aaaa", "aaaf"));

File: problem-set/Problem-Petya-And-Strings/Petya-And-Strings.cpp
#include <iostream>
#include <string>
 
using namespace std;
 
int main()
{
    string s1, s2;
    cin >> s1 >> s2;
    
    for (int x = 0; x < s1.size(); x++) {
        int num1 = tolower(s1[x]);
        int num2 = tolower(s2[x]);
        
        if (num1 < num2) {
            cout << -1 << endl;
            return 0;
        } else if (num1 > num2) {
            cout << 1 << endl;
            return 0;
        }
    }
    
    cout << 0 << endl;
 
    return 0;
}